source/includes/cpp-library-reference/_simulator.html.md

```diff
@@ -17,7 +17,6 @@ demonstrates how these are called during training. Optionally, one may also over
 | ---               | ---         |
 | `brain`           |  The simulators Brain object. |
 | `name`            |  The simulators name. |
-| `objective_name`  |  The name of the current objective for an episode. |
```

Comment:
What will be the value of `objective_name` for v2 programs? If it's totally going away, we need to update the SDK and add some useful error messages -- there are definitely some programs that use `objective_name` floating around.